content/pt-br/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: blog
+title: "Redirecionamento do registry de imagens k8s.gcr.io para registry.k8s.io - O que você precisa saber"
+date: 2023-03-10T17:00:00.000Z
+slug: image-registry-redirect
+---
+
+**Autores**: Bob Killen (Google), Davanum Srinivas (AWS), Chris Short (AWS), Frederico Muñoz (SAS
+Institute), Tim Bannister (The Scale Factory), Ricky Sadowski (AWS), Grace Nguyen (Expo), Mahamed
+Ali (Rackspace Technology), Mars Toktonaliev (independent), Laura Santamaria (Dell), Kat Cosgrove
+(Dell)
+
+
+Na segunda-feira, dia 20 de março, o registry de imagens k8s.gcr.io [vai ser redirecionado para o registry da comunidade](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/), **registry.k8s.io**.
+
+
+## O que você precisa saber sobre essa mudança
+- Na segunda-feira, dia 20 de março, o tráfego do registry antigo k8s.gcr.io vai ser redirecionado para
+registry.k8s.io com o objetivo final de encerramento do k8s.gcr.io.
+- Se você está executando em um ambiente restrito, e aplica políticas rígidas de acesso a nomes de domínios
+ou endereços IPs limitado a k8s.gcr.io, **o pull de imagens não vai funcionar** depois que o k8s.gcr.io começar
+a redirecionar para o novo registry.
+- Um pequeno subconjunto de clientes que não seguem o padrão não lidam com redirecionamentos HTTP, e neste caso eles
+precisam ser apontados diretamente para registry.k8s.io.
+- O redirecionamento é um paliativo para ajudar os usuários a fazer essa troca. O registro obsoleto k8s.gcr.io será desativado em algum momento. **Por isso, atualize seus manifestos o mais rápido possível para apontar para registry.k8s.io**.
+- Se você mantém seu próprio registry de imagens, você pode copiar as imagens que você precisa para reduzir o tráfego.
+ao registry da comunidade.
+
+Se você acha que vai ser impactado, ou gostaria de saber mais sobre essa mudança, leia mais abaixo.
+
+## Como eu posso checar se eu vou ser afetado?
+Para testar a conectividade ao registry.k8s.io (e saber se consegue fazer baixar as imagens), pode executar o
```

Comment:
"(e saber se consegue baixar as imagens)" instead of "(e saber se consegue fazer baixar as imagens)"